Optics/Wave Propagation/src/Wave.js

```diff
@@ -35,42 +35,47 @@ class Wave {
 		}
 	}
 
-	amplitude(x, t) {
-		// Onde sinusoidale
-/*		let k = 2;
-		let w = 10;
-		let w = 0.2;
-		return Math.sin(k*x - w * t);*/
-
-		// Onde progressive
-/*		let v = 10;
-		return 3*Math.exp(-Math.pow(x - v*t + this.offset, 2));*/
-
-
-		// Phénomène de dispersion
-/*		let dk = 1;
-		let dw = 5;
-		let k = 10;
-		let w = 20;
-		// return Math.cos(k/2*x-w/2*t) + Math.cos((k + dk)/2*x-(w + dw)/2*t);
-		return { a : Math.cos(dk/2*x-dw/2*t), b : Math.cos((k + dk/2)*x-(w + dw/2)*t)};*/
-
-		// Stationnaire
-		//t = 0;
-		let k = 2;
-		let w = 2;
-		return { a: Math.sin(k*x - w*t), b : Math.sin(k*x + w*t), c : 2 * Math.cos(w*t) * Math.cos(k*x) };
-
-		// Battements
-/*		let v1 = 0.1;
-		let v2 = 1.2;
-		let k = 10;
-		return Math.cos(k*x) * Math.cos((v1 + v2) * 2*Math.PI*t);*/
-
-		// Signal modulé
-/*		let v1 = 0.1;
-		let v2 = 1.2;
-		let b = 3;
-		return Math.cos(x + v1*t) + b/2 * Math.cos(x + (v1 + v2)*t) + b/2 * Math.cos(x + (v1 - v2)*t);*/
+	amplitude(c, x, t) {
```

Comment:
- [ ] Rename variable `c` to a more descriptive variable like `waveType`.
- [ ] Remove the parts `c == 1`, `c == 2`, ... which are not very usefull